docs/installation/installation.rst

```diff
@@ -30,6 +30,9 @@ There are several ways to install Augur, ordered from least to most sophisticate
 
    .. group-tab:: PyPI
 
+      .. warning::
+         Installing other Python packages after Augur may cause dependency incompatibilities. If this happens, re-install Augur using the command in step 1.
```

Comment:
_non-blocking_

I think this should be less common with newer Pip versions which detect the issue. It might be more common when mixing Python packages installed via both Pip and Conda?

Reply:
Ah, I see now that Pip [detects but _doesn't prevent_](https://github.com/nextstrain/augur/issues/1358#issuecomment-1906967971). orz

Reply:
Yeah, unfortunate 😕